### PR TITLE
added scales

### PR DESCRIPTION
**Pull request by clsource**
Based on https://github.com/xiphonics/picoTracker/issues/39

### CODE REVIEW

**Comment by clsource** [comment]
closes #119

File: sources/Application/Model/Project.cpp [hunk 1]
```diff
@@ -27,6 +28,11 @@ tempoNudge_(0)
 	this->Insert(wrap) ;
 	Variable *transpose=new Variable("transpose",VAR_TRANSPOSE,0) ;
 	this->Insert(transpose) ;
+    
+    // from: https://github.com/xiphonics/picoTracker/blob/master/sources/Application/Model/Project.cpp
```
Comment (djdiskmachine) [inline sources/Application/Model/Project.cpp:32]: I'd prefer if we keep this info in the corresponding .h and .cpp, it's not necessary to have it here.

File: sources/Application/Model/Project.cpp [hunk 2]
```diff
@@ -59,6 +65,13 @@ Project::~Project() {
 	delete instrumentBank_ ;
 } ;
 
+// from: https://github.com/xiphonics/picoTracker/blob/master/sources/Application/Model/Project.cpp
```
Comment (djdiskmachine) [inline sources/Application/Model/Project.cpp:68]: I'd prefer if we keep this info in the corresponding .h and .cpp, it's not necessary to have it here.

File: sources/Application/Views/PhraseView.cpp [hunk 3]
```diff
@@ -206,6 +207,12 @@ void PhraseView::updateCursorValue(ViewUpdateDirection direction,int xOffset,int
 	}
 	if ((c)&&(*c!=0xFF)) {
 		int offset=offsets_[col_+xOffset][direction] ;
+		// from: https://github.com/xiphonics/picoTracker/blob/master/sources/Application/Views/PhraseView.cpp
```
Comment (djdiskmachine) [inline sources/Application/Views/PhraseView.cpp:210]: I'd prefer if we keep this info in the corresponding .h and .cpp, it's not necessary to have it here.

File: sources/Application/Views/PhraseView.cpp [hunk 4]
```diff
@@ -1178,6 +1186,27 @@ void PhraseView::DrawView() {
 
     drawMap() ;
    	drawNotes() ;
+
+	// Draw Current Scale
```
Comment (djdiskmachine) [inline sources/Application/Views/PhraseView.cpp:1190]: Please no commits containing commented-out code

File: sources/Application/Views/ProjectView.cpp [hunk 5]
```diff
@@ -114,6 +115,17 @@ ProjectView::ProjectView(GUIWindow &w,ViewData *data):FieldView(w,data) {
 	position._y+=1 ;
 	UIIntVarField *f2=new UIIntVarField(position,*v,"transpose: %3.2d",-48,48,0x1,0xC) ;
 	T_SimpleList<UIField>::Insert(f2) ;
+	
+	// from: https://github.com/xiphonics/picoTracker/blob/master/sources/Application/Views/ProjectView.cpp
```
Comment (djdiskmachine) [inline sources/Application/Views/ProjectView.cpp:119]: I'd prefer if we keep this info in the corresponding .h and .cpp, it's not necessary to have it here.

**Review by djdiskmachine — CHANGES_REQUESTED** [review]

**Comment by djdiskmachine** [comment]
Closed in https://github.com/djdiskmachine/LittleGPTracker/pull/157

**Pull request closed without merging**